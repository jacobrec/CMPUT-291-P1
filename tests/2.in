1
navras
drowssap
2
Lover
Boy
Lover
Girl








2
Lover
Boy
Other
Girl




2
Other
Girl
Lover
Girl
7
2

