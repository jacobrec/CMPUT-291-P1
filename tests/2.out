Please sign in to use App:
1) login
2) exit
Username: Password: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Partner 1 - First Name: Partner 1 - Last Name: Partner 2 - First Name: Partner 2 - Last Name: Enter details for Lover Boy
Birth Date: Birth Place: Address: Phone Number: Enter details for Lover Girl
Birth Date: Birth Place: Address: Phone Number: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Partner 1 - First Name: Partner 1 - Last Name: Partner 2 - First Name: Partner 2 - Last Name: Enter details for Other Girl
Birth Date: Birth Place: Address: Phone Number: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Partner 1 - First Name: Partner 1 - Last Name: Partner 2 - First Name: Partner 2 - Last Name: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Please sign in to use App:
1) login
2) exit

